docs/ref/settings.txt

```diff
@@ -146,7 +146,7 @@ The :setting:`APPEND_SLASH` setting is only used if
 CACHES
 ------
 
-Default::
```

Comment:
These double colon changes before code blocks are incorrect as we lose formatting in the rendered version of the docs.